monix-execution/jvm/src/main/scala/monix/execution/internal/ScheduledExecutors.scala

```diff
@@ -28,14 +28,17 @@ private[execution] object ScheduledExecutors {
     * an underlying Java `ScheduledExecutorService`.
     */
   def scheduleOnce(
-    self: Scheduler,
-    executor: ScheduledExecutorService)(initialDelay: Long, unit: TimeUnit, r: Runnable): Cancelable = {
+    executor: Scheduler,
+    scheduler: ScheduledExecutorService
+  )(
+    initialDelay: Long, unit: TimeUnit, r: Runnable
+  ): Cancelable = {
     if (initialDelay <= 0) {
       executor.execute(r)
```

Comment:
I didn't realize initially that bug is on this line, because the actual diff looks like just rename

Reply:
Probably worth noting in commit message/release notes/etc. that it was triggered by using nonpositive delays

Reply:
Indeed. I think it was the naming that made this confusing, hence the rename.